src/plugins/recordTypes/loanout/forms/default.jsx

```diff
@@ -0,0 +1,75 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Cols,
+    Col,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+  } = configContext.recordComponents;
+
+  return (
```

Comment:
This whole form probably needs to be rearranged to make the tab order make sense, but it's fine for now. I'll play with it after merging.